TalkableSDK.podspec

```diff
@@ -0,0 +1,36 @@
+
+Pod::Spec.new do |spec|
+
+  spec.name         = "TalkableSDK"
+  spec.version      = "1.0"
```

Comment:
Lets have the same version as SDK: `1.4.11`

Reply:
Ok, i'l change version

Reply:
Do we need to bump version? To ```1.4.12```

Reply:
No, there is no changes in SDK

Reply:
Ok, I setted version  `1.4.11`